Translation corrections (#88)
07th-mod/higurashi-console-arcs: Update/some_006.txt

//r何もできない…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "何もできない…。",

> 			NULL, "I couldnn't do anything...", Line_Normal);

Reviewer: ```suggestion
			NULL, "I couldn't do anything...", Line_Normal);
```

	ClearMessage();

//r大好きな家族のみんながバラバラになっていくのを、私はただ、眺め見ることしかできない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大好きな家族のみんながバラバラになっていくのを、私はただ、眺め見ることしかできない。",

> 			NULL, "I could only look on as everyone in the family I love falls apart.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I could only look on as everyone in the family I love fell apart.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>夏美の祖母</color>", NULL, "<color=#f5e6d3>Natsumi's grandmother</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310800075", 256, TRUE);
	OutputLine(NULL, "「………祟りは、本当にあるんだよ」",
			NULL, "\".........There really is a curse.\"", Line_Normal);
	ClearMessage();

//rぼそりとかすかに、おばーちゃんの呟き。まるで、その言葉が祟りそのもののような陰鬱な響きを持っていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ぼそりとかすかに、おばーちゃんの呟き。まるで、その言葉が祟りそのもののような陰鬱な響きを持っていた。",

> 			NULL, "Grandma was muttering faintly, her voice melancholic. As if that word were a "curse" in and of itself.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Grandma was muttering faintly, her voice melancholic. As if that word were a \"curse\" in and of itself.", Line_Normal);
```

//rそう言うと、おとーさんはホッとしたように笑った。…きっと、おとーさんも私と同じで、…心の底から信じてはいないんだろうけど……。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そう言うと、おとーさんはホッとしたように笑った。…きっと、おとーさんも私と同じで、…心の底から信じてはいないんだろうけど……。",

> 			NULL, "As I said that, Dad smiled with relief. ...But I'm sure that deep down... he wasn't any more convinced than me......", Line_Normal);

Reviewer: ```suggestion
			NULL, "As I said that, Dad smiled with relief. ...But I'm sure that deep down... he didn't believe that any more than I did......", Line_Normal);
```

	ClearMessage();

	DrawScene("black", 1000 );

//rそれに、…おばーちゃんは本当に、雛見沢の災害のことをすぐに忘れてくれるかな……？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それに、…おばーちゃんは本当に、雛見沢の災害のことをすぐに忘れてくれるかな……？",

> 			NULL, "Besides... would Grandma ever forget about the disaster in Hinamizawa that soon......?", Line_Normal);

Reviewer: ```suggestion
			NULL, "Besides... would Grandma ever forget about the disaster in Hinamizawa that quickly......?", Line_Normal);
```

//rあんなおばーちゃん、顔も見たくない…！　一緒の空気も吸いたくなんかない！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あんなおばーちゃん、顔も見たくない…！　一緒の空気も吸いたくなんかない！",
			NULL, "I don't want to see that face of hers...! I don't want to breathe the same air as her!", Line_Normal);
	ClearMessage();

//r……そうだ。家に不吉な空気を運んできたのは、私の生まれ育った興宮の隣、雛見沢での不可解な事件のニュースではなく、おばーちゃんの奇妙な態度なんだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "……そうだ。家に不吉な空気を運んできたのは、私の生まれ育った興宮の隣、雛見沢での不可解な事件のニュースではなく、おばーちゃんの奇妙な態度なんだ。",

> 			NULL, "......Right. This sinister air was carried into my house not by the news by the news of the disaster in Hinamizawa, but Grandma's bizarre behavior.", Line_Normal);

Reviewer: ```suggestion
			NULL, "......Right. This sinister air was carried into my house not by the news of the disaster in Hinamizawa, but Grandma's bizarre behavior.", Line_Normal);
```
